components/footer/_footer.scss

```diff
@@ -1,13 +1,46 @@
 .footer {
   padding: 2rem 0;
   margin-top: 2rem;
-  background-color: $color-light-grey;
-  color: $color-very-dark-grey;
+  background-color: $color-very-dark-grey;
+  color: $color-light-grey;
+
+  // ul li { not required as list--bare is available
```

Comment:
Remove commented out code. Also, we shouldn't be using element selectors if possible.